packages/blocks/src/page-block/default/utils.ts

```diff
@@ -122,13 +122,25 @@ export async function downloadImage(model: BaseBlockModel) {
 export async function copyImgToClip(model: BaseBlockModel) {
   const url = await getUrlByModel(model);
   assertExists(url);
-  const data = await fetch(url);
-  const blob = await data.blob();
-  await navigator.clipboard.write([
-    new ClipboardItem({
-      [blob.type]: blob,
-    }),
-  ]);
+
+  const newImage = new Image();
+  newImage.src = url;
+  newImage.onload = () => {
+    const canvas = document.createElement('canvas');
+    canvas.width = newImage.width;
+    canvas.height = newImage.height;
+    const ctx = canvas.getContext('2d');
+    ctx?.drawImage(newImage, 0, 0);
```

Comment:
This will create a "deep copy" of the image. This can be merged for fast iteration, but I hope @darkskygit can take a look at what the best practice could be.

Reply:
In any case, images that can be successfully loaded by img can be obtained by fetch() (this means that the corresponding link complies with cross-domain rules)If we use canvas screenshots, there will be slight differences in the original binary of each pasted image , which will invalidate the hash-based deduplication of the blob manager, and will also generate additional encoding and decoding overhead

Reply:
simply say, it can be written like this:
``` js
fetch(url).then(r => r.blob()).then(blob => {
    // do something with blob
})
```